weir-cmd/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM weirgroup/build-image
+
+LABEL "version"="1.0.0"
+LABEL com.github.actions.name="GitHub Action for Generic Shell Actions"
+LABEL com.github.actions.description="Pass it anything to run anything."
+LABEL com.github.actions.icon="shield"
```

Update to the command icon for clarity.


```suggestion
LABEL com.github.actions.icon="command"
```